### PR TITLE
Chainer: Simplify n_step_lstm test

### PR DESCRIPTION
**Pull request by dido1998**

### CODE REVIEW

**Comment by take-cheeze** [comment]
Jenkins, test this please!

File: tests/chainer_tests/functions_tests/rnn_tests/test_function_n_step_lstm.py [hunk 1]
```diff
@@ -375,290 +311,12 @@ def check_forward(
...
-    # return cuda.cupy.random.randint(-2, 2, shape).astype('f')
-    return cuda.cupy.random.uniform(-1, 1, shape).astype('f')
-    # return cuda.cupy.ones(shape).astype('f')
-
-
-@testing.parameterize(*testing.product({
-    'use_cudnn': ['always', 'auto', 'never'],
-}))
-@attr.cudnn
-class TestNStepLSTMDropout(unittest.TestCase):
```
Comment (take-cheeze) [inline tests/chainer_tests/functions_tests/rnn_tests/test_function_n_step_lstm.py]: Is removing test for `dropout = 0.3` intended?

**Comment by take-cheeze** [comment]
The jenkins failure seems to be unrelated

File: tests/chainer_tests/functions_tests/rnn_tests/test_function_n_step_lstm.py (hunk 1 above)
Comment (dido1998) [reply tests/chainer_tests/functions_tests/rnn_tests/test_function_n_step_lstm.py]: Yes, you're right, I had removed it by mistake, I have added it back now.

**Comment by take-cheeze** [comment]
ChainerCI, test this please!

**Comment by take-cheeze** [comment]
Jenkins, test this please!

**Comment by dido1998** [comment]
Is this failure related to this PR?

**Comment by take-cheeze** [comment]
Seems jenkins error doesn't relate to this PR so don't mind since pfn ci is working fine

**Comment by take-cheeze** [comment]
Jenkins, test this please!

**Comment by take-cheeze** [comment]
Jenkins, test this please!

**Comment by take-cheeze** [comment]
Jenkins, test this please!

**Comment by take-cheeze** [comment]
ChainerCI, test this please!

**Comment by dido1998** [comment]
@take-cheeze can this be merged?

**Comment by take-cheeze** [comment]
Seems to be OK. Thanks for reminding me.

**Comment by take-cheeze** [comment]
ChainerCI, test this please!

**Pull request merged**